stem/model.py

```diff
@@ -295,6 +295,16 @@ def synchronise_geometry(self):
         # get the complete geometry
         self.__get_geometry_from_geo_data(self.gmsh_io.geo_data)
 
+    def set_mesh_size(self, element_size:float):
+        """Set the element size to dimension (m).
```

Comment:
can you format the docstring correctly in this whole file? Like how you have in kratos_model_io.py